Bot.py

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[ ]:
```

Comment:
It's better to remove unnecessary comments like this

Reply:
Done